src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
```

Comment:
4. **Error 4: Unused parameter in method**  
   The `subtract` method declares an unused parameter `c`. If it's not necessary, the parameter should be removed.